models/consistency.go

```diff
@@ -70,7 +83,7 @@ func assertCount(t *testing.T, bean interface{}, expected int) {
 		"Failed consistency test, the counted bean (of type %T) was %+v", bean, bean)
 }
 
-func (user *User) checkForConsistency(t *testing.T) {
+func checkForConsistencyUser(user *User, t *testing.T) {
```

Comment:
I'd say that is one of two possible naming schemes.
The other would have been
```suggestion
func checkForUserConsistency(user *User, t *testing.T) {
```
Same below then.
What do you say about this approach?